doc/newfeat.texi

```diff
@@ -59,6 +59,13 @@ The following have been made obsolete, see their documentation for more detail:
 
 @itemize @bullet
 
+@item Libraries for literate data analysis.
+This includes @code{ess-noweb}, @code{ess-swv}, and related
+functionality like @code{Rnw-mode}. Users are encouraged to switch to
+one of several other packages that deal with these modes. For example,
+polymode @url{https://github.com/polymode/polymode} or markdown-mode
```

Comment:
Users should be directed to [poly-R](https://github.com/polymode/poly-R/) and polymode [doc page](https://polymode.github.io/). Just installing poly-R from MELPA should be enough for most users. 

Also maybe mention that `ess-noweb-mode` can be still used by explicitly requiring them.

Reply:
I added those links.